lms/djangoapps/grades/tests/test_transformer.py

```diff
@@ -330,6 +330,16 @@ def test_course_version_collected_in_split(self):
         block_structure = get_course_blocks(self.student, blocks[u'course'].location, self.transformers)
         self.assertIsNotNone(block_structure.get_xblock_field(blocks[u'course'].location, u'course_version'))
 
+    def test_grading_policy_collected(self):
+        blocks = self.build_course_with_problems()
+        block_structure = get_course_blocks(self.student, blocks[u'course'].location, self.transformers)
+        self.assert_collected_transformer_block_fields(
+            block_structure,
+            blocks[u'course'].location,
+            self.TRANSFORMER_CLASS_TO_TEST,
+            grading_policy_hash=-5366073452054513847,
```

Comment:
Instead of providing a magic number here, could you define it as a variable (class or method, whichever seems most appropriate) with a comment explaining how you got it? Even if it's as simple as "This is the resulting has of the default grading policy".

Also, how hard would it be to change the grading policy slightly, then confirm that the previously calculated hash no longer matches?